### PR TITLE
Absorb column projections in ``ReadCSV``

### PR DESCRIPTION
**Pull request by rjzamora**
Simple follow-up to https://github.com/dask-contrib/dask-expr/pull/247 for `ReadCSV`.

### CODE REVIEW

File: dask_expr/io/csv.py [hunk 1]
```diff
@@ -37,10 +37,7 @@ def _ddf(self):
 
     @property
     def _meta(self):
-        meta = self._ddf._meta
-        if self.columns:
-            return meta[self.columns[0]] if self._series else meta[self.columns]
-        return meta
+        return self._ddf._meta
```
Comment (rjzamora) [inline dask_expr/io/csv.py:40]:
```suggestion
        meta = self._ddf._meta
        if self.operand("columns") is not None:
            return meta[self.columns[0]] if self._series else meta[self.columns]
        return meta
```

File: dask_expr/io/tests/test_io.py [hunk 2]
```diff
@@ -282,12 +282,16 @@ def test_to_parquet(tmpdir, write_metadata_file):
         df2.to_parquet(tmpdir, overwrite=True)
 
 
-def test_combine_similar(tmpdir):
+@pytest.mark.parametrize(
+    "fmt,read_func,read_cls",
```
Comment (phofl) [inline dask_expr/io/tests/test_io.py:286]: Personally, I like spaces in between these arguments, but no blocker

**Review by phofl — APPROVED** [review]

**Comment by phofl** [comment]
thx

File: dask_expr/io/tests/test_io.py (hunk 2 above)
Comment (rjzamora) [reply dask_expr/io/tests/test_io.py:286]: Didn't even realize I was allowed to leave spaces here, but that makes sense :)

**Pull request merged**